src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/ManageContent/IKeyStatisticService.cs

```diff
@@ -0,0 +1,29 @@
+﻿#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Requests;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Services.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces.ManageContent;
+
+public interface IKeyStatisticService
+{
+    Task<Either<ActionResult, KeyStatisticDataBlockViewModel>> CreateKeyStatisticDataBlock(
+        Guid releaseId, KeyStatisticDataBlockCreateRequest request);
+
+    Task<Either<ActionResult, KeyStatisticDataBlockViewModel>> UpdateKeyStatisticDataBlock(
+        Guid releaseId,
+        Guid keyStatisticId,
+        KeyStatisticDataBlockUpdateRequest request);
+
+    Task<Either<ActionResult, Unit>> Delete(Guid releaseId, Guid keyStatisticId);
+
+    Task DeleteAssociatedKeyStatisticDataBlock(Guid dataBlockId);
+
+    Task<Either<ActionResult, List<KeyStatisticViewModel>>> Reorder(
+        Guid releaseId,
+        Dictionary<Guid, int> newKeyStatisticOrder);
```

Comment:
I've switched it to a `List`. This has meant that I've had to switch back to throwing an ArgumentException rather than returning a `ValidationErrorMessage` for this case - this is necessary for this to work with `.ForEach()`

Reply:
I'm also going to create a topic to switch other instances where we reorder from a dictionary to a list. I think it would be good for this to be consistent.